projects/dsp-ui/src/lib/action/pipes/string-transformation/truncate.pipe.spec.ts

```diff
@@ -1,21 +1,62 @@
 import { TruncatePipe } from './truncate.pipe';
+import { Component } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
 
 describe('TruncatePipe', () => {
     let pipe: TruncatePipe;
+    let snippet: string;
 
     beforeEach(() => {
         pipe = new TruncatePipe();
+        snippet = 'The quick brown fox jumps over the lazy dog.';
     });
 
     it('create an instance', () => {
         expect(pipe).toBeTruthy();
     });
 
     it('should truncate after 15 characters', () => {
-        expect(pipe.transform('The quick brown fox jumps over the lazy dog.', ['15'])).toEqual('The quick brown...');
+        const truncatedSnippet = pipe.transform(snippet, 15);
+        expect(truncatedSnippet).toEqual('The quick brown...');
     });
 
     it('should truncate after 19 characters and add an exclamation mark at the end', () => {
-        expect(pipe.transform('The quick brown fox jumps over the lazy dog.', ['19', '!'])).toEqual('The quick brown fox!');
+        const truncatedSnippet = pipe.transform(snippet, 19, '!');
+        expect(truncatedSnippet).toEqual('The quick brown fox!');
+    });
+
+    it('should truncate after 20 characters by default', () => {
+        const truncatedSnippet = pipe.transform(snippet);
+        expect(truncatedSnippet).toEqual('The quick brown fox ...');
+    });
+
+    it('should support the limit argument in the template', () => {
+        @Component({
+          template: `{{ text | dspTruncate:4 }}`,
+        })
+        class App {
+          text = 'This is my string';
+        }
+
+        TestBed.configureTestingModule({declarations: [App, TruncatePipe]});
+        const fixture = TestBed.createComponent(App);
+        fixture.detectChanges();
+
+        expect(fixture.nativeElement.textContent).toBe('This...');
```

Comment:
very small remark: I prefer `toEqual` (equivalence) since `toBe` checks for identify. 

https://stackoverflow.com/questions/22413009/jasmine-javascript-testing-tobe-vs-toequal

But it does not matter in case of primitive values.

Reply:
I agree, has been changed to `toEqual` in 365750d